### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by ilanolkies**

### CODE REVIEW

**Comment by vojtechsimetka** [comment]
I would suggest for other rebranding we extract the CSS and components into NPM package that we can just import and centrally manage.

Other than that good job @ilanolkies !

**Comment by ilanolkies** [comment]
@vojtechsimetka sure! Let’s do it!

File: package.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "bootstrap": "^4.3.1",
```
Comment (bguiz) [inline package.json]: This comment is unrelated to this PR, more of a general one for the project:

My preference for dependencies declared in `package.json` is to lock down your versions. For example, this line could be:

- `"bootstrap": "4.3.1",` --> only `4.3.1`
- `"bootstrap": "~4.3.1",` --> `4.3.1` or `4.3.2`, `4.3.3`, etc (if more patch versions get released)

Right now, if you run the installation, you could get `4.3.1`, `4.4.0`, or `4.4.1` ... basically anything until `5.x.x` gets released.

Comment (bguiz) [reply package.json]: ... and bonus: Much less diff-noise in `package-lock.json` and `yarn.lock` files.

File: public/assets/js/vendor/bootstrap.min.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*!
```
Comment (bguiz) [inline public/assets/js/vendor/bootstrap.min.js]: Why is this file being committed,when it is also being installed as a dependency in `package.json`?

It would be better if to just choose one option:

- Commit this file --> Nothing else needs to be done
- Install via package.json --> Add a `postinstall` run script to copy this file across from within `node_modules`, and `.gitignore` this one

File: public/index.html [hunk 3]
```diff
@@ -1,15 +1,78 @@
...
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
+    <!-- build:js -->
+    <script src="assets/js/main.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/select2@4.0.12/dist/js/select2.min.js"></script>
```
Comment (bguiz) [inline public/index.html]: Why not make this dependency consistent with the rest, and install it via npm, or commit the file within the vendor folder?

File: public/assets/js/vendor/bootstrap.min.js (hunk 2 above)
Comment (ilanolkies) [reply public/assets/js/vendor/bootstrap.min.js]: I removed the npm dependecy.

File: public/assets/img/RIFOS RNS MANAGER.html [hunk 4]
```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<!-- saved from url=(0073)file:///X:/PROYECTOS/RIF/RNS_Manager/V6/TEMPLATE/admin-resolution-qr.html -->
```
Comment (bguiz) [inline public/assets/img/RIFOS RNS MANAGER.html]: You sure that you want to commit with this comment?

Also why is this html file in `public/assets/img`?

File: public/assets/img/RIFOS RNS MANAGER.html [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+      <meta http-equiv="x-ua-compatible" content="ie=edge">
+      <meta name="description" content="RIFOS RNS MANAGER">
+      <meta name="keywords" content="">
+      <meta name="author" content="RIFOS">
+      <title>RIFOS RNS MANAGER</title>
+      <!-- favicons -->
+      <link rel="apple-touch-icon" sizes="180x180" href="file:///X:/PROYECTOS/RIF/RNS_Manager/V6/TEMPLATE/assets/favicons/apple-touch-icon.png">
+      <link rel="icon" type="image/png" sizes="32x32" href="file:///X:/PROYECTOS/RIF/RNS_Manager/V6/TEMPLATE/assets/favicons/favicon-32x32.png">
+      <link rel="icon" type="image/png" sizes="16x16" href="file:///X:/PROYECTOS/RIF/RNS_Manager/V6/TEMPLATE/assets/favicons/favicon-16x16.png">
+      <link rel="manifest" href="file:///X:/PROYECTOS/RIF/RNS_Manager/V6/TEMPLATE/assets/favicons/site.webmanifest">
```
Comment (bguiz) [inline public/assets/img/RIFOS RNS MANAGER.html]: This `href` value is unlikely to work, when run externally.

Is this file necessary for the faucet?

File: public/assets/img/RIFOS RNS MANAGER.html [hunk 6]
```diff
@@ -1,240 +0,0 @@
-<!DOCTYPE html>
```
Comment (bguiz) [inline public/assets/img/RIFOS RNS MANAGER.html]: :+1:

File: public/index.html [hunk 7]
```diff
@@ -63,7 +63,7 @@
     -->
     <!-- build:js -->
     <script src="assets/js/main.js"></script>
-    <script src="https://cdn.jsdelivr.net/npm/select2@4.0.12/dist/js/select2.min.js"></script>
+    <script src="vendor/select2.min.js"></script>
```
Comment (bguiz) [inline public/index.html]: :+1:
